### ISSUE TITLE
Add `aria-label` to inputs

### ISSUE THREAD

**Issue by nickbristow** [wojtekmaj/react-time-picker#21]
For accessibility reasons, each of the inputs generated should have a label. 

Adding `aria-label='hour'`to the hour input, and one for minute, and PM/AM should be on by default, or give us the option to enable them.

**Comment by wojtekmaj** [comment]
Will introduce:

* `hourAriaLabel`
* `minuteAriaLabel`
* `secondAriaLabel`
* `amPmAriaLabel`

**Comment by emilyuhde** [comment]
The x to clear icon button and the clock icon button should both have props to add an aria-label as well. Anything clickable needs an aria-label.

**Comment by wojtekmaj** [comment]
Closed via 318845a03b0918686ebbe99936b723469c84daa1
